src/module.c

```diff
@@ -8155,8 +8317,11 @@ void moduleNotifyKeyspaceEvent(int type, const char *event, robj *key, int dbid)
      *
      * In order to do that we increment the execution_nesting counter, thus
      * preventing postExecutionUnitOperations (from within moduleFreeContext)
-     * from propagating commands from CB. */
-    server.execution_nesting++;
+     * from propagating commands from CB.
+     *
+     * This is a special case where we need to increase 'execution_nesting'
+     * but we do not want to update the cached time so we can not use enterExecutionUnit */
+    enterExecutionUnit(0, 0);
```

Comment:
I don't get this comment, we can't use enterExecutionUnit and then we immediately call it?

Reply:
comment is outdated :smile: we fixed the problem that prevented the use of this function (by passing an argument), and forgot to update or delete the comment.

Reply:
Good catch, will update.

Reply:
Updated